code/modules/reagents/chemistry/reagents/misc.dm

```diff
@@ -520,4 +551,4 @@
 	id = "triplepiss"
 	description = "Ewwwwwwwww."
 	reagent_state = LIQUID
-	color = "#857400"
\ No newline at end of file
+	color = "#857400"
```

Comment:
So none of these above get any tastes? I mean maybe they aren't used but that sounds like a different problem that needs to be solved